SPECS/openssh/regress-test-future-cert-fix.patch

```diff
@@ -0,0 +1,44 @@
+From d62a8caee3324abc5e921269d975db986585c7d3 Mon Sep 17 00:00:00 2001
+From: Thomas Crain <thcrain@microsoft.com>
+Date: Tue, 5 Jan 2021 11:13:12 -0800
+Subject: [PATCH] Patch regression tests with future dates
+
+The future is now, at least according to the regression tests. Tests expecting
+2020-01-01 to be in the future are invalid now, so we give those tests new
+future dates. 
+
+Patch adapted from https://github.com/openssh/openssh-portable/commit/ff31f15773ee173502eec4d7861ec56f26bba381
+---
+ regress/cert-hostkey.sh | 2 +-
+ regress/cert-userkey.sh | 2 +-
+ 2 files changed, 2 insertions(+), 2 deletions(-)
+
+diff --git a/regress/cert-hostkey.sh b/regress/cert-hostkey.sh
+index 3ce77796..74d5a530 100644
+--- a/regress/cert-hostkey.sh
++++ b/regress/cert-hostkey.sh
+@@ -248,7 +248,7 @@ test_one() {
+ test_one "user-certificate"	failure "-n $HOSTS"
+ test_one "empty principals"	success "-h"
+ test_one "wrong principals"	failure "-h -n foo"
+-test_one "cert not yet valid"	failure "-h -V20200101:20300101"
++test_one "cert not yet valid"	failure "-h -V20300101:20320101"
```

Comment:
Might as well go for the next millennium to never have to go back to this. Or at least to be sure we won't be around to worry about it.;)

Reply:
Sure thing :)

I don't know the date of my own death, so I'll just eyeball it as the late 90s.